core/lib/zksync_core/src/block_reverter/mod.rs

```diff
@@ -221,7 +221,7 @@ impl BlockReverter {
         tracing::info!("opening DB with state keeper cache...");
         let sk_cache = RocksdbStorage::builder(self.state_keeper_cache_path.as_ref())
             .await
-            .expect("Failed initializing state keeper cache");
+            .expect("Failed initializing state keeper");
```

Comment:
Nit: Not sure this change is warranted; the log above talks about the state keeper cache.